_sass/_reset.scss

```diff
@@ -65,6 +65,13 @@ a img {
     border: 0;
 }
 
+dd{
```

Comment:
Might it be better to find where the `dl` & `dt` styles are being unset and remove those lines / update as needed?  

Additionally, I personally quite like this idea, but are there any pages on the site where this styling might muck something up?  (i would hope not, but we should be sure.)

Thanks!

Reply:
They're unset a few lines above in this file, but I didn't want to change that as it also sets some other general properties for them. 

`dl`s are used in a couple articles where this will improve their look.
The only other place with a `dl` is [events.html](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/events.html) to assist with screenreader compatibility, but it has class names on the `dt` and `dd`s, which should override this setting.